packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -89,7 +92,19 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
     this.stopRecordingEvents = record({
       emit: (event) => {
         const eventString = JSON.stringify(event);
-        const shouldSplit = shouldSplitEventsList(this.events, eventString, this.maxPersistedEventsSize);
+        // Send the first two recorded events immediately
```

Comment:
I don't quite follow how this is executing for the first 2 events instead of just the first event. What's the case?

Reply:
At this point in the code, we haven't added the second emitted event to `this.events` yet, so there is the first emitted event in `this.events`, then the second `event`, and we send the two of those together. We end up never adding the second emitted event to `this.events` at all